index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vangelis</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body>
+    <div class="header">
+      <!-- Navigation Bar w/ Welcome message and Hamburger -->
+      <nav>
+        <ul>
+          <li>Welcome</li>
+          <li>
+            <a href="#"><img src="/images/Union.png" alt="toggle-switch" /></a>
```

Comment:
- [ ] [ OPTIONAL ] To ensure that your images work perfectly across all browsers, kindly avoid adding a `/` at the beginning of the image url. Consider using `"images/myimage.png"` instead of  `"/images/myimage.png"`